lambdas/reset-identity/src/main/java/uk/gov/di/ipv/core/resetidentity/ResetIdentityHandler.java

```diff
@@ -112,7 +114,16 @@ public Map<String, Object> handleRequest(ProcessRequest event, Context context)
 
             // Make sure we do this before deleting the credentials!
             String userName = null;
-            if (isUserInitiated) {
+            CriResponseItem f2fRequest = criResponseService.getFaceToFaceRequest(userId);
```

Comment:
Just thinking do we _need_ to retrieve the pending F2F request here? Could we use `lambdaInput` in the journey map with a specific state for this route (from the F2F pending page)? something like
```
RESET_IDENTITY_USER_INITIATED_F2F:
  response:
    type: process
    lambda: reset-identity
    lambdaInput:
      isUserInitiatedF2f: true
  events:
    next:
      ...
```

Reply:
Hmm ok that's not ideal, let's stick with the current approach